### ISSUE TITLE
React component names changed, menus no longer work

### ISSUE THREAD

**Issue by davepagurek** [rsms/fb-mac-messenger#308]
In <a href="https://github.com/rsms/fb-mac-messenger/blob/master/website/app/main.js#L160-L170">these lines</a>, we try to find the settings gear by looking for the `MessengerMasterViewHeader` component. Unfortunately, this component no longer seems to exist, when looking through the React debugger:

![screen shot 2016-07-16 at 1 54 09 am](https://cloud.githubusercontent.com/assets/5315059/16893054/3da5ecb4-4af8-11e6-820a-82b70457d966.png)

Instead, we seem to just have components called `k`, `m`, `o`, and other generic names, so the app's javascript can't find the components it's looking for any more.

This is likely the cause of https://github.com/rsms/fb-mac-messenger/issues/305 as well.

**Comment by rsms** [comment]
Yup. If anyone can find some time to fix this and submit a pull request, that would be fantastic.

**Comment by davepagurek** [comment]
Do you think these components are "important" enough that they won't change and we should build new react component queries around them? Or is a different query (possibly based on the `role` attribute) a better way to grab these elements?
